### PR TITLE
{tools}[intel/2020a] pyiron v0.3.0 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* #11231
* #11232
* #11234
* #11240
* #11242

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3152.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1de5ec9c729ad4fc1c19eca48598747c for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/pyiron/pyiron-0.3.0-intel-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    ('scikit-learn', '0.23.1', versionsuffix),
+    ('yaff', '1.6.0', versionsuffix),
+    ('TAMkin', '1.2.6', versionsuffix),
+    ('QuickFF', '2.2.4', versionsuffix),
+]
+
+check_ldshared = True
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/p/pyiron/pyiron-0.3.0-intel-2020a-Python-3.8.2.eb:35]:
```suggestion
```

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 525.105.17, Python 3.6.8
See https://gist.github.com/SebastianAchilles/9e38b203be04863c3c736d7ebc80b65d for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/06009809687131b29e7d7d482de8065c for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**